test/service/test_response.py

```diff
@@ -2565,7 +2653,7 @@ def test_matrix_cell_count(self):
             file_format = file['format']
             count = file['matrix_cell_count']
             actual_counts[file_format] += 0 if count is None else count
```

Comment:
Worried that we're conflating None and 0.

```suggestion
            if count is not None:
                actual_counts[file_format] += count
```